build.gradle.kts

```diff
@@ -63,6 +63,9 @@ dependencies {
     implementation("io.ktor", "ktor-client-cio", ktorVersion)
     implementation("io.ktor", "ktor-client-json", ktorVersion)
     implementation("io.ktor", "ktor-serialization", ktorVersion)
+    implementation("io.ktor", "ktor-server-netty", ktorVersion)
```

Comment:
Don't use netty when we are already using CIO

Reply:
Alright I forgot about that